### PR TITLE
:book: Correcting a typo in uas.md file 

### PR DESCRIPTION
**Pull request by pm-harshad-mane**
:book: Correcting a typo in uas.md file
- Need to use accId instead of accountId.

### CODE REVIEW

File: ads/uas.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Note that the `width` and `height` mentioned should be maximum of the width-high
 <amp-ad
     width="300" height="250"
     type="uas"
-    json='{"accId": "132109", "adUnit": "10002912", "sizes": [[300, 250]], "targetings": {"country": ["India", "USA"], "car": "Civic"}, "locLat": "12.24", "locLon": "24.13", "locSrc": "wifi", "pageURL": "http://mydomain.com"}'>
+    json='{"accId": "132109", "adUnit": "10002912", "sizes": [[300, 250]], "targetings": {"country": ["India", "USA"], "car": "Civic"}, "locLat": "12.24", "locLon": "24.13", "locSrc": "1", "pageURL": "http://mydomain.com"}'>
```
Comment (lannka) [inline ads/uas.md]: I didn't find the supported value for `locSrc`. Is `1` supported? `wifi` might actually make sense for "location source"

Comment (pm-harshad-mane) [reply ads/uas.md]: Hello @lannka ,

Thank you for your suggestion.
PubMatic UAS tag needs to pass the value for locSrc parameter as per a pre-defined map of values thus we are changing it from "wifi" to "1".

Comment (lannka) [reply ads/uas.md]: OK, that sounds good. But if you folks a the maintainer of PubMatic, I suggest you provide the full map in the documentation as well.

**Comment by pm-harshad-mane** [comment]
Hello @lannka and @zhouyx ,
When can we merge these changes?

**Comment by pm-harshad-mane** [comment]
Thank you @lannka

**Pull request merged**